customer-commons/src/main/java/no/unit/nva/customer/model/ApplicationDomain.java

```diff
@@ -0,0 +1,35 @@
+package no.unit.nva.customer.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.net.URI;
+import java.util.Arrays;
+import nva.commons.core.SingletonCollector;
+
+public enum ApplicationDomain {
+    
+    NVA(URI.create("https://nva.org"));
```

Comment:
We should change this -> but I don't know to what the final domain will be…at least nva.unit.no for now

Reply:
done